runelite-api/src/main/java/net/runelite/api/IndexedSprite.java

```diff
@@ -25,119 +25,39 @@
 package net.runelite.api;
 
 /**
- * Represents an indexed sprite.
+ * Represents an paletted sprite.
  */
 public interface IndexedSprite
 {
 	/**
-	 * Gets the pixels contained by the sprite.
-	 *
-	 * @return the sprite pixels
+	 * The bitmap of this sprite. Each value is an index into {@link #getPalette()}.
+	 * 0 is transparent
 	 */
 	byte[] getPixels();
-
-	/**
-	 * Sets the pixels contained by the sprite.
-	 *
-	 * @param pixels the new sprite pixels
-	 */
 	void setPixels(byte[] pixels);
 
 	/**
-	 * Gets the color palette for the sprites pixels.
-	 *
-	 * @return the color palette
+	 * The color palette in RGB. The zero index is unused.
 	 */
 	int[] getPalette();
-
-	/**
-	 * Sets the color palette for the sprites pixels.
-	 *
-	 * @param palette the new color palette
-	 */
 	void setPalette(int[] palette);
 
-	/**
-	 * Gets the offset of the sprite along the x-axis.
-	 *
-	 * @return the x-axis offset
-	 */
 	int getOffsetX();
-
-	/**
-	 * Sets the offset of the sprite along the x-axis.
-	 *
-	 * @param offsetX new x-axis offset
-	 */
 	void setOffsetX(int offsetX);
 
-	/**
-	 * Gets the offset of the sprite along the y-axis.
-	 *
-	 * @return the y-axis offset
-	 */
 	int getOffsetY();
-
-	/**
-	 * Sets the offset of the sprite along the y-axis.
-	 *
-	 * @param offsetY new y-axis offset
-	 */
 	void setOffsetY(int offsetY);
 
-	/**
-	 * Gets the width of the sprite.
-	 *
-	 * @return the width
-	 */
 	int getWidth();
-
-	/**
-	 * Sets the width of the sprite.
-	 *
-	 * @param width the new width
-	 */
 	void setWidth(int width);
 
-	/**
-	 * Gets the original width of the sprite.
-	 *
-	 * @return the width
-	 */
-	int getOriginalWidth();
-
-	/**
-	 * Sets the original width of the sprite.
-	 *
-	 * @param originalWidth the width
-	 */
-	void setOriginalWidth(int originalWidth);
 
-	/**
-	 * Gets the height of the sprite.
-	 *
-	 * @return the height
-	 */
 	int getHeight();
-
```

Comment:
this line shouldnt be removed

Reply:
It is a paired interface method, so it should be removed